_posts/0000-01-07-amanagrawal.md

```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![evil-raccooon](https://cloud.githubusercontent.com/assets/16547949/25400793/b1104eee-29c1-11e7-89e2-3c99f9ea1941.jpg)
 {: .center}
 
-Caption here
+Yaaay!!
```

Comment:
@amanagrawal Maybe you should make the caption a little more sinister since the raccoon looks really mean.

Reply:
@beardofedu - evil raccoons rule.